architecture/additional_concepts/authentication.adoc

```diff
@@ -206,13 +206,136 @@ When using a service account as an OAuth client:
 $ oc sa get-token <serviceaccount name>
 ----
 
-* `redirect_uri` must match an annotation on the service account that has the
-prefix `serviceaccounts.openshift.io/oauth-redirecturi`. For example,
-`serviceaccounts.openshift.io/oauth-redirecturi.one`.
 * To get `WWW-Authenticate` challenges, set an
 `serviceaccounts.openshift.io/oauth-want-challenges` annotation on the service
 account to *true*.
 
+* `redirect_uri` must match an annotation on the service account:
```

Comment:
It looks like the bulk of this new content (from here through to *OAuth Server Metadata*) is about this bullet, is that right? I'm assuming that because of the colon. It seems like a lot of info to have trying to be part of a bullet (currently it doesn't have `+`s so it's not really part of the bullet, if that was the intention).

What about making a new heading here... Either just bold or a real `====` subheading within "Service Accounts as OAuth Clients"? Maybe called "Redirect URIs" or "Redirect URIs in Service Accounts".